torchmetrics/functional/text/bert.py

```diff
@@ -530,14 +533,22 @@ def bert_score(
             A path to the user's own local csv/tsv file with the baseline scale.
         baseline_url:
             A url path to the user's own  csv/tsv file with the baseline scale.
+        predictions:
+            Either an iterable of predicted sentences or a `Dict[str, torch.Tensor]` containing `input_ids` and
+            `attention_mask` `torch.Tensor`.
+            This argument is deprecated in v0.7 and will be removed in v0.8. Use `preds` instead.
+        references:
+            Either an iterable of target sentences or a `Dict[str, torch.Tensor]` containing `input_ids` and
+            `attention_mask` `torch.Tensor`.
+            This argument is deprecated in v0.7 and will be removed in v0.8. Use `target` instead.
```

Comment:
```suggestion
        predictions:
            .. deprecated:: v0.7
                This argument is deprecated in favor of  `preds` and will be removed in v0.8.
        references:
            .. deprecated:: v0.7
                This argument is deprecated in favor of  `preds` and will be removed in v0.8.
```
just need to check therendering